src/main/java/com/cinema/model/Ticket.java

```diff
@@ -17,9 +16,9 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Column(name = "ticket_id")
     private Long id;
-    @ManyToOne (fetch = FetchType.LAZY)
+    @ManyToOne
```

Comment:
let's remove fetchType in MovieSession as well

Reply:
I did it in today's HW - it will be a challenge to merge now two HWs...